editor/state.js

```diff
@@ -303,6 +303,21 @@ export function isSidebarOpened( state = false, action ) {
 	return state;
 }
 
+export function editorWidth( state = null, action ) {
+	const layout = document.querySelector( '.editor-layout__content' );
```

Comment:
I mean more about cases where we may dispatch this from other places of the app, like the sidebar toggle. We shouldn't have to repeat the code there, I think.